### PR TITLE
Updated functions for vectors

### PR DESCRIPTION
**Pull request by anujshandillya**
# Added `.emplace_back()` function for vectors
### Description

Added function for vectors `.emplace_back()` in vectors section of C++ docs.

# Added `.emplace_back()` function for vectors

### Type of Change

- Adding a new entry
- ~~Editing an existing entry (fixing a typo, bug, issues, etc)~~
- ~~Updating the documentation~~

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by avdhoottt** [comment]
@anujshandillya Thanks for collaborating but we can't accept this PR. Please don't create a PR if you are not assigned to an issue. First, Ask the maintainer to assign you the issue and then proceed with your solution and create a PR. You can choose any issue listed [here](https://github.com/Codecademy/docs/issues).

**Comment by anujshandillya** [comment]
@avdhoottt this is not an issue, I have just updated the document with new entries:
we use push_back() as well as emplace_back() and both adds element at back of vector but both are different in working.
Please look into it.

**Comment by avdhoottt** [comment]
Sorry for the Confusion, I'm reviewing this PR.

File: content/cpp/concepts/vectors/vectors.md
Comment (avdhoottt) [inline content/cpp/concepts/vectors/vectors.md:153]:
```suggestion
This method also pushes elements to the back of a vector, but instead of creating a temporary object it directly creates an object in the vector itself.
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@anujshandillya I have suggested some minor errors, please correct them.

**Comment by anujshandillya** [comment]
@avdhoottt hey I have looked into suggested changes and updated it.

**Comment by anujshandillya** [comment]
@avdhoottt hey please approve the changes, I have looked into it.

**Review by avdhoottt — APPROVED** [review]
PR is ready for the second review.

**Comment by anujshandillya** [comment]
@SSwiniarski could you take a look and review this PR.

**Review by caupolicandiaz — APPROVED** [review]
@anujshandillya @avdhoottt LGTM

**Pull request merged**
